integrations/PinterestTag/browser.js

```diff
@@ -201,9 +201,10 @@ export default class PinterestTag {
     }
 
     /*
-    Step 3: In case both of the above stated cases fail, will mark the event as "custom"
+    Step 3: In case both of the above stated cases fail, will send the event name as it is.
+          This is going to be reflected as "unknown" event in pinterest tag dashboard.
    */
-    return ['custom'];
+    return [event];
```

Comment:
We have to add unitTestcase to cover this.

Reply:
Right now we dont have in JS-SDK